examples/green-house-migration/legacy/greenhouse/batch/users.py

```
@@ -0,0 +1,37 @@
+
+
+class UsersProcessor(BaseProcessor):
+    """Processor for fetching user data and related info."""
+
+    entity = "users"
+
+    def fetch(self):
+        users = fetch_all_from_api("users")
+
+        for _user in users:
+            user_id = user.get("id")
```

Ruff (0.12.2)

Undefined name `user`

(F821)